### ISSUE TITLE
i have some questions about your paper on arvix

### ISSUE THREAD

**Issue by ColaWithoutIcee** [yhao-z/T2LR-Net#1]
Hi, yhao-z

 I read your recent article on accelerated magnetic resonance imaging on arvix. I have a small question for you. In the discussion section of the article, you discussed the impact of the hyperparameter \zeta in the loss function on the performance of the entire network. You found that the network can achieve the best performance when the loss function does not impose a symmetric closed solution constraint (\zeta = 0). So is the CNN part still exploring sparse transformation? Can it be further optimized with networks of other architectures?

**Comment by yhao-z** [comment]
Hi,
As we discussed in the arxiv paper, we think the proposed network not only utilize the low rank, but also the implict deep image prior from CNN. So I think the CNN part is not specific at learning a sparse prior. Maybe some more complex features that help the reconstruction could be learned. 

But our network still focuses on the low rank property, so i think if other priors, e.g., sparsity or sth. effective, could be incorperated with our proposed transformed low rank to improve the performance.

hope it helps.

**Comment by ColaWithoutIcee** [comment]
Thx, I got it.
Have a good day and hope your paper will be accepted soon.
